docs/operations/basic-cluster-tuning.md

```diff
@@ -90,7 +90,7 @@ Tuning the cluster so that each Historical can accept 50 queries and 10 non-quer
 
 #### Segment Cache Size
 
-For better query performance, do not allocate segment data to a Historical in excess of the system free memory.  When `free system memory` is greater than or equal to `druid.segmentCache.locations`, the more segment data the Historical can be held in the memory-mapped segment cache.
+For better query performance, do not allocate segment data to a Historical in excess of the system free memory.  When `free system memory` is greater than or equal to `druid.segmentCache.locations`, the Historical's memory-mapped segment cache can be held in memory, rather than being paged to disk.
```

Comment:
I think the new wording is a little misleading. Memory-mapping happens even if free system memory is less, but not all segments would be memory mapped.

[This section](https://druid.apache.org/docs/latest/design/historical/#loading-and-serving-segments-from-cache) has a clearer explanation of the segment cache. We can reuse it.

Reply:
Yeah, it makes sense to link the other doc here. But I think we should get rid of the second sentence altogether as there are other problems with it too.

However, if we do want to keep that sentence, it could be a lot simpler, like

```
The Historical uses free system memory to cache segments.
```

This sentence is easy to understand and clearly implies that the higher the free memory, the higher the number of segments that can be cached. The details are captured in the other doc anyway.